tests/components/wiz_light/__init__.py

```diff
@@ -0,0 +1 @@
+"""Tests for the WiZ Light integration integration."""
```

Comment:
I will be surprised if The Powers That Be approve this PR without any unit tests for the integration.

Reply:
That's correct but there are almost no documentation how and what to test.
I will start to dig in the pytesing of HASS.

Reply:
Please use our scaffold script to generate example code including tests for the config flow using best practices patterns.

Reply:
I already used the script to generate the pattern. But overall there will no "test" generated.
The only docs for the testing are [here](https://developers.home-assistant.io/docs/development_testing#writing-tests-for-integrations).

Reply:
It does generate test code. Please try again.

Reply:
No, it doesn't in the current dev branch via devcontainer. I also tried it with an "clean" demo integration. It also doesn't create a test. Only the __init__ will be created.

I'll try to reproduce and create a test manually.

Reply:
When I run this on dev branch (and answer the questions):
```
python -m script.scaffold integration
```
I get this output:
```
Integration code has been generated

Added the following files:
- homeassistant/components/wiz_light/manifest.json
- homeassistant/components/wiz_light/config_flow.py
- homeassistant/components/wiz_light/__init__.py
- homeassistant/components/wiz_light/const.py

Added the following tests:
- tests/components/wiz_light/test_config_flow.py

The next step is to look at the files and deal with all areas marked as TODO.
```

![scaffold_wiz_light](https://user-images.githubusercontent.com/3181692/103409041-410bbb00-4b65-11eb-8000-277f4b6f4eef.png)